### PR TITLE
Strings for 4.0

### PR DESCRIPTION
**Pull request by boek**

### CODE REVIEW

File: templates/focus-ios.xliff [hunk 1]
```diff
@@ -145,6 +201,26 @@
         <source>Stop</source>
         <note>Accessibility label for the stop button</note>
       </trans-unit>
+      <trans-unit id="Cancel">
+        <source>Cancel</source>
+        <note>Label on button to cancel edits</note>
+      </trans-unit>
+      <trans-unit id="Disabled">
+        <source>Disabled</source>
```
Comment (Delphine) [inline templates/focus-ios.xliff]: The generic "disabled/enabled" here won't work for any of the languages that use gender (such as French, Spanish, German, etc).
Unfortunately we'll have to know each time what those strings refer to (a masculine or feminine word?) in order to properly localize in many of our languages.

File: templates/focus-ios.xliff [hunk 2]
```diff
@@ -145,6 +201,26 @@
...
+      <trans-unit id="Done">
+        <source>Done</source>
+        <note>Label on button to complete edits</note>
+      </trans-unit>
+      <trans-unit id="Edit">
+        <source>Edit</source>
+        <note>Label on button to allow edits</note>
+      </trans-unit>
+      <trans-unit id="Enabled">
+        <source>Enabled</source>
```
Comment (Delphine) [inline templates/focus-ios.xliff]: Same as above: The generic "disabled/enabled" here won't work for any of the languages that use gender (such as French, Spanish, German, etc).
Unfortunately we'll have to know each time what those strings refer to (a masculine or feminine word?) in order to properly localize in many of our languages.v

File: templates/focus-ios.xliff [hunk 3]
```diff
@@ -269,13 +396,19 @@
         <source>Settings</source>
         <note>Title for settings screen</note>
       </trans-unit>
+      <trans-unit id="Settings.searchLabel">
+        <source>Search Engine</source>
+        <note>Label for the search engine in the search screen</note>
+      </trans-unit>
       <trans-unit id="Settings.searchSection">
         <source>SEARCH ENGINE</source>
-        <note>Title for the search engine row</note>
```
Comment (Delphine) [inline templates/focus-ios.xliff]: Wondering why we're dropping the localization note in this case? is this expected?

File: templates/focus-ios.xliff [hunk 4]
```diff
@@ -269,13 +396,19 @@
...
       </trans-unit>
       <trans-unit id="Settings.searchTitle">
         <source>Search Engine</source>
-        <note>Title for the search engine selection screen</note>
```
Comment (Delphine) [inline templates/focus-ios.xliff]: Same as above: wondering why we're dropping the localization note in this case?

Comment (boek) [reply templates/focus-ios.xliff]: We removed this string from the app. Replacing it with `Settings.searchTitle2`

**Comment by boek** [comment]
Made a new PR #31

File: templates/focus-ios.xliff (hunk 4 above)
Comment (flodolo) [reply templates/focus-ios.xliff]: Ehm… https://github.com/mozilla-mobile/focus-ios/issues/395#issuecomment-326502215

Comment (boek) [reply templates/focus-ios.xliff]: @flodolo good to know. I can remove them and do another export if you'd like? cc: @Delphine

Comment (flodolo) [reply templates/focus-ios.xliff]: Likely good enough to remove them now in code, so they're fixed with the next export and not forgotten.

**Pull request closed without merging**